src/components/MapboxMap/MapboxMap.vue

```diff
@@ -163,6 +248,27 @@ function initMap(accessToken: string) {
   map.on("moveend", () => {
     saveMapState();
   });
+
+  map.on("zoom", () => {
+    const currentZoom = map.getZoom();
+
+    switch (true) {
+      case currentZoom >= 17:
+        map.setLayoutProperty(CLUSTERS_LAYER_ID, VISIBILITY_PROPERTY, "none");
+        map.setLayoutProperty(UNCLUSTERED_LAYER_ID, VISIBILITY_PROPERTY, "visible");
+        break;
+
+      case currentZoom < 14:
```

Comment:
Is this an actual use case? If we're zoomed out pretty much, we see no markers at all?

Reply:
I think somewhere we need to restrict the max zoom out because our markers + color overlay aren't visible on a certain zoom level. Still have to figure this out